File: QuestoesBeecrowd-Iniciante/1009.c
#include <stdio.h>
#include <stdlib.h>

double total(double s, double v)
{
    double porcentagem;
    porcentagem = s + (v * 0.15);
    return (porcentagem);
}

int main()
{

    char nome;
    double salario, vendas, final;

    scanf("%s", &nome);
    scanf("%lf", &salario);
    scanf("%lf", &vendas);
    final = total(salario, vendas);
    printf("TOTAL = R$ %.2f\n", final);

    return 0;
}

File: QuestoesBeecrowd-Iniciante/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let name = String(lines.shift());
let salarioFixo = Number(lines.shift());
let vendas = Number(lines.shift());

console.log(`TOTAL = R$ ${resultado(salarioFixo,vendas)}`)


function resultado(salario, vendas) {
  let valorFinal = vendas * 0.15 + salario;
  let final = valorFinal.toFixed(2);
  return final;
}

